src/components/common/header/PageHeader.tsx

```diff
@@ -0,0 +1,61 @@
+import Tippy from '@tippyjs/react'
+import React, { useContext } from 'react'
+import { NavLink } from 'react-router-dom'
+import { DOCUMENTATION, SERVER_MODE, URLS } from '../../../config'
+import { mainContext } from '../../common/navigation/NavigationRoutes'
+import { ReactComponent as Question } from '../../../assets/icons/ic-help-outline.svg'
+
+export interface PageHeaderType {
+    headerName: string
+    buttonText?: string
+    onClickCreateButton?: () => void
+    serverModeName?: string
```

Comment:
Do we need this? can this check be implemented in showCreateButton?

Reply:
done